setup.cfg

```diff
@@ -17,7 +17,8 @@ project_urls =
 [options]
 install_requires =
     six>=1.14.0
-    click>=7.0.0
+    click>=8.0.0; python_version >= '3.7'
+    click>=7.0.0; python_version < '3.6'
```

Comment:
What are the consequences of using click 7 in older versions of Python? Are there some features that don't work on those Python versions?